src/sumTwo/index.js

```diff
@@ -0,0 +1,18 @@
+const twoSum = (list, target) => {
+
+  for (let i = 0; i < list.length; i++) {
+    const first = list[i];
+    
+    for (let j = 0; j < list.length; j++) {
+      const second = list[j];
+      if (i === j ) continue;
+
+      const sum = first + second;
+      if ( sum === target ) {
+        return [i, j];
+      }
+    }
+  }
+}
+
+export default twoSum;
```

Comment:
Is there a solution with better time complexity than O(N^2)?

Reply:
Certainly, I would figure it out, as soon as I can